content/rest-api/filter.md

```diff
@@ -160,6 +160,139 @@ To get the products that were updated during the last 4 days, you can use the fo
+
+#### Example
+To get the product models of the `winter_collection` category, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"categories":[{"operator":"IN","value":["winter_collection"]}]}
+```
+
+### On completeness
+
+To filter product models on their completeness, use the `completeness` product property. You will also need to provide a `scope` value to specify on which channel you want to filter with the completeness.
+Here are the allowed operators you can use to filter by completeness as well as the corresponding type of value expected in the `search` query parameter.
+
+| Operator | Allowed value type | Filter description |
+| ----------------- | -------------- | ------------------ |
+| `AT LEAST COMPLETE` | no value | Only returns product models that have at least one variant product fully complete on the given channel for the given set of locales |
+| `ALL COMPLETE` | no value |  Only returns product models that have all his variant products fully complete on the given channel for the given set of locales |
+
+#### Examples
+To get the product models that are 100% complete for the `ecommerce` channel, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"completeness":[{"operator":"COMPLETE","scope":"ecommerce"}]}
+```
+
+To get the product models that have at least one variant product 100% complete on both the `en_US` and `fr_FR` locales for the `ecommerce` channel, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"completeness":[{"operator":"AT LEAST COMPLETE","value":100,"locales":["en_US","fr_FR"],"scope":"ecommerce"}]}
+```
+
+### On family
+
+To filter product models on families, use respectively the product model property `family`.
+Here are the allowed operators you can use to filter on this property as well as the corresponding type of value expected in the `search` query parameter.
+
+| Operator | Allowed value type | Filter description |
+| ----------------- | -------------- | ------------------ |
+| `IN` | array of existing family | Only returns product models that are in the given family |
+| `NOT IN`  | array of existing family | Only returns product models that are not in the given family |
+| `EMPTY` | no value | Only returns product models that have no family |
+| `NOT EMPTY` | no value | Only returns product models that have a family |
+
+#### Examples
+To get the product models that are in the `camcorders` family, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"family":[{"operator":"IN","value":["camcorders"]}]}
+```
+
+To get the product models that are not in the `camcorders` and `digital_cameras` family, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"family":[{"operator":"NOT IN","value":["camcorders","digital_cameras"]}]}
+```
+
+### On creation or update date
+
+To filter product models on creation or update date, use respectively the product property `created` and `updated`.
```

Comment:
use the product properties 'created' and 'updated', respectively

Reply:
Fix